references/sigproc/new_IMGQ_metrics.md

```diff
@@ -6,15 +6,16 @@ Being able to measure the quality of the image displayed by our device is of cru
 
 However, quantifying the quality of an ultrasound imaging system is difficult as no objective criteria has yet been fully developed and accepted for the evaluation of clinical image quality. For the moment, the evaluation of the quality of a clinical image is generally performed subjectively by a specialist, even if a few techniques providing relevant measures have been developed in the past years. 
```

Comment:
Great job 😉 
Can you cite the references in the text ?
Actually we have a plugin to handle citations, you can look at IMGQ_metrics.md to see how it works

I think you can elaborate a bit more the description of the metrics - the scheme for the L-alpha is very instructive, it would be nice to have something like that for each metric, e.g. mathematical definition / expression + an illustration

It would be interesting to add more details about tissue mimicking phantoms - how to make some homemade ones ?
What would be the requirements for a 2-axis table ? (e.g. what precision on the position of the target do we need to achieve ? Do we need to monitor the position automatically ? ...) What metrics can we measure with it ?
What about the contract, L-alpha and so on ? What would we need to measure that ?